internal/app/singularity/actions.go

```diff
@@ -3,7 +3,7 @@
 // LICENSE.md file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
-// +build linux
+// +build singularity_runtime
```

Comment:
Using `singularity_runtime` build tag to include files for when we are building on Linux (currently) and including the `runtime` component in the binary.